### ISSUE TITLE
API to update time against a tag

### ISSUE THREAD

**Issue by quaintdev** [traggo/server#104]
First of all, thanks a lot for creating traggo. 

Is it possible to update time against a particular tag by calling certain API? I did not find any documentation related to this. If it is not currently supported any pointers where I should start if I want to implement this functionality.

Basically I already have certain tools in place which can track my screen time on both phone and laptop. I would want them to automatically update their respective time against tags in traggo.

Thanks!

**Comment by jmattheis** [comment]
Yes, you can use the graphql api. See https://github.com/traggo/server/blob/master/schema.graphql#L28
